pkg/daemon/ceph/client/mgr_test.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	exectest "github.com/rook/rook/pkg/util/exec/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEnableModule(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithOutputFile = func(debug bool, actionName, command, outputFile string, args ...string) (string, error) {
```

Comment:
To test the retry logic, I'd suggest the test set the `moduleEnableWaitTime` to 0 and then this mock method could return false at least once.

Reply:
Thanks for the suggestion. Added a different method `TestEnableModuleRetries` to test the retry.